docs/guidance/map/milestones.md

```diff
@@ -0,0 +1,177 @@
+# Milestones
+
+Milestones can be included within the planning, tender, contract and contract implementation sections. 
+
+## Planning
+
+The planning milestones block is used for two types of milestones:
+ * Key events in the planning stage, for example, the preparation of an environmental impact assessment, the approval to proceed with a project, or the date of a public consultation. 
+ * Anticipated milestones during the contract implementation stage, for example, the date by which goods delivery of the goods is required.
+
+If during the planning stage you have information about tender process milestones, then you
+populate the tender milestones block instead.
+
+## Tender
+
+The tender milestones block is used to describe two types of milestone:
+  * Key dates in the tender and award stages which are not covered by other fields, for example, the date by which procuring entity will respond to enquiries.
+  * Anticipated milestones during the contract implementation stage, for example, the date by which goods need to be delivered.
+
+## Contracts and implementation
```

Comment:
Split into two headings.

Reply:
I changed this to "Contract" as the description just describe contracts and not implementation here

Reply:
Doesn't this bullet point describe the use of `contract/implementation/milestones`?

> * Any events related to the delivery of the contract, for example, the agreed date by which goods will be delivered.

My understanding is the `contract/milestones` is just for events related to the signing of the contract.

Reply:
Oh yes, you are right! Fixed!